lib/wallets/wallet/wallet_mixin_interfaces/cw_based_interface.dart

```diff
@@ -218,9 +218,10 @@ mixin CwBasedInterface<T extends CryptonoteCurrency> on CryptonoteWallet<T>
   }
 
   Future<void> _refreshTxData() async {
+    // Calling the function too quickly causes crashes.
+    await Future<void>.delayed(const Duration(seconds: 1));
```

Comment:
This was an experiment, and crashes still occured, less frequently - but this may have nothing to do with the root cause.